src/pluto/pluto.py

```diff
@@ -478,14 +478,14 @@ def generate_graph(v1, v2):
                     "pfirm15_flag",
                 ]
                 x = [(v1[i] / v2[i] - 1) * 100 for i in y]
-                diff=[v1[i] - v2[i]  for i in y]
+                diff = [v1[i] - v2[i] for i in y]
                 real_v1 = [v1[i] for i in y]
                 real_v2 = [v2[i] for i in y]
                 hovertemplate = "<b>%{x}</b> %{text}"
                 text = []
                 for n in range(len(y)):
                     text.append(
-                        "Diff: {} Current: {} Prev: {}".format(
+                        "Diff: {} | Current: {} | Prev: {}".format(
```

Comment:
Nice addition and compartmentalizes the text better

Reply:
oh shoot I didn't mean to add this lol. Do you like it? I decided against it but definitely open to discussion